app/emoji-store/service.js

```diff
@@ -24,7 +42,10 @@ export default Ember.Service.extend({
   seedEmoji() {
     for (let name of Object.keys(emojilib.lib)) {
       let {char, category, keywords} = emojilib.lib[name];
-      let emoji = {name: humanize(name), char};
+      let emoji = {
+        name: (category === 'flags')?countryName(name.toUpperCase()):humanize(name),
```

Comment:
I think I would add this conditional logic to the `humanize` function. It might seem a little odd but I think we could consider country code -> country name mappings as part of "humanizing".